script/frontier/install.sh

```diff
@@ -27,6 +27,7 @@ module load cray-python/3.9.13.1
 module load hdf5/1.14.0
 module load ninja/1.10.2
 module load tmux/3.2a
+module unload darshan-runtime/3.4.0
```

Comment:
```suggestion
module unload darshan-runtime
```
Handle future cases when the default `darshan-runtime` is a different version.

Also, move this change to `frontier.jinja` (and as Dom suggested, `summit.jinja`). `install.sh` will be overwritten when `make_dockerfiles.py` is executed.